solutions/day_5/solution.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+from functools import reduce
+
+
+TEXT_FILE_LOCATION = "input.txt"
+BATCH_SIZE = 3
+
+CRATE_HOLDER = {1: [], 2: [], 3: [], 4: [], 5: [], 6: [], 7: [], 8: [], 9: []}
+
+
+def whitespace_helper(val: str) -> bool:
```

Comment:
This function is incorrect. Because if the `True` branch is not executed, then it returns `None`, not `False`.